pkg/services/oauthtoken/oauth_token_test.go

```diff
@@ -226,16 +238,26 @@ func setupOAuthTokenService(t *testing.T) (*Service, *FakeAuthInfoStore, *social
 	socialConnector := &socialtest.MockSocialConnector{}
 	socialService := &socialtest.FakeSocialService{
 		ExpectedConnector: socialConnector,
+		ExpectedAuthInfoProvider: &social.OAuthInfo{
+			UseRefreshToken: true,
+		},
 	}
 
-	authInfoStore := &FakeAuthInfoStore{}
+	authInfoStore := &FakeAuthInfoStore{
+		ExpectedOAuth: &login.UserAuth{
```

Comment:
I'm not a bit fan of setting this here, it's a bit misleading then in the tests, what token we are using (ex: `TestService_TryTokenRefresh_NoRefreshToken` does not actually use the token defined in the test itself)

Reply:
I removed all the initialization of the authInfoStore